CHANGELOG.md

```diff
@@ -4,14 +4,38 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 2.4.0 (2021-07-26)
+This release adds new experimental features since the 2.3 release.
+The experimental features in this release are:
+* APIs for chunk manipulation across data nodes in a distributed
+hypertable setup. This includes the ability to add a data node and move
+chunks to the new data node for cluster rebalancing.
+* The `time_bucket_ng` function, a newer version of `time_bucket`. This function
```

Comment:
@Loquacity Should we explain that ng stands for **N**ew **G**eneration?

Reply:
Yep, absolutely.